dm/tests/sync_collation/conf/dm-task-increment.yaml

```diff
@@ -0,0 +1,105 @@
+---
+name: task-name-placeholder
+task-mode: incremental
+is-sharding: false
+meta-schema: "dm_meta"
+# enable-heartbeat: true
+heartbeat-update-interval: 1
+heartbeat-report-interval: 1
+clean-dump-file: false
+
+target-database:
+  host: "127.0.0.1"
+  port: 4000
+  user: "root"
+  password: ""
+
+mysql-instances:
+  - source-id: "mysql-replica-01"
+    meta:
+      binlog-name: binlog-name-placeholder-1
+      binlog-pos: binlog-pos-placeholder-1
+      binlog-gtid: binlog-gtid-placeholder-1
+    block-allow-list:  "instance"
+    route-rules: [ "collation-table-rules-1",collation-table-rules-1-2,"collation-schema-rules-1","collation-table-rules-increment-1","collation-schema-rules-increment-1"]
+    mydumper-config-name: "global"
+    loader-config-name: "global"
+    syncer-config-name: "global"
+
+  - source-id: "mysql-replica-02"
+    meta:
+      binlog-name: binlog-name-placeholder-2
+      binlog-pos: binlog-pos-placeholder-2
+      binlog-gtid: binlog-gtid-placeholder-2
+    block-allow-list:  "instance"
+    route-rules: [ "collation-table-rules-2",collation-table-rules-2-2,"collation-schema-rules-2","collation-table-rules-increment-2","collation-schema-rules-increment-2"]
+    mydumper-config-name: "global"
+    loader-config-name: "global"
+    syncer-config-name: "global"
+
+block-allow-list:
+  instance:
+    do-dbs: ["sync_collation*"]
+
+routes:
+  collation-table-rules-1:
+    schema-pattern: "sync_collation"
+    target-schema: "sync_collation"
+    table-pattern: "t1"
+    target-table: "t"
+  collation-table-rules-1-2:
```

Comment:
no need to add this route, it has no effect 🤔

Reply:
`collation-table-rules-1-2`‘s original intention is upstream `sync_collation.t2` to downstream `sync_collation.t2`.